__tests__/app.test.js

```diff
@@ -29,24 +30,65 @@ describe("GET /api/topics", () => {
         });
       });
   });
-  test("404: responds with not found for wrong endpoint", () => {
+  // test("404: responds with not found for wrong endpoint", () => {
+  //   return supertest(app)
+  //     .get("/api/something")
+  //     .expect(404)
+  //     .then((response) => {
+  //       expect(response.body).toEqual({ error: "Not found" });
+  //       expect(response.statusCode).toBe(404);
+  //     });
+  // });
+});
+
+describe("/api", () => {
+  test("200 - responds with json file containing list of endpoints", () => {
     return supertest(app)
-      .get("/api/something")
-      .expect(404)
+      .get("/api")
+      .expect(200)
       .then((response) => {
-        expect(response.body).toEqual({ error: "Not found" });
-        expect(response.statusCode).toBe(404);
+        expect(jsonfile);
       });
   });
 });
 
-describe("/api", () => {
-  test.only("200 - responds with json file containing list of endpoints", () => {
+describe("GET /api/articles/:article_id", () => {
+  test("200 - Responds with a status code of 200", () => {
+    return supertest(app).get("/api/articles/1").expect(200);
+  });
+  // test("Responds with an object containing the correct article", () => {
```

Comment:
I don't understand why this is greyed out. You are expecting (hopefully) your server to respond with an object with a key of article with the value of an object that matches what you have here.